CHANGES.rst

```diff
@@ -4,9 +4,16 @@
 New Tools and Services
 ----------------------
 
+
 API changes
 -----------
 
+mast
+^^^^
+
+- Handle a MAST URI string as input for ``Observations.get_cloud_uri`` and a list of MAST URIs as input for
+  ``Observations.get_cloud_uris``. [#3193]
+
```

Comment:
fyi: I'll move this section down below as we keep this section to highlight potentially breaking API changes rather than for additions/enhacements.